application/scheduled-quarkus/src/test/java/webapp/tier/schedule/CallRandomPublshScheduleTest.java

```diff
@@ -1,30 +1,36 @@
 package webapp.tier.schedule;
 
-import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import javax.inject.Inject;
 
 import org.junit.jupiter.api.Test;
 
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
 class CallRandomPublshScheduleTest {
 
+	@Inject
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=6004366)